### DISCUSSION TITLE
Mobile doesn’t all for filter to ALlL open PRs

### DISCUSSION THREAD

**Discussion by afitzgerald** [community/community#20233] [Mobile]
![image](https://user-images.githubusercontent.com/100173/178103102-90dc27fe-c969-47ed-98fa-2e0e784cd388.png)

**Comment by elisealix22** [comment]
Hi @afitzgerald , thank you so much for reaching out! For context, the default scope for pull requests from Home is ones you've created (the first option). Are you trying to see all open pull requests for a repository? Or is there another subset of open pull requests you're searching for?

You might be able to add a [shortcut](https://github.blog/changelog/2022-01-13-introducing-shortcuts-on-github-mobile/) for the list you'd like to see on Home!  Those options are pretty customizable.

**Reply by afitzgerald** [reply]
A shortcut is no different than the default filter criteria, there is no option not to choose from this filter option and/or all option.

**Comment by kronn** [comment]
Just to chime in: I also want to see all PRs (open and in an organization, but that can be filtered already).

I tried with the shortcuts, but there I am limited to the options visible in the screenshot above: "Created, Assigned, Mentioned, Review Requested". Most of the PRs I want to review are not in those buckets and I do not want to change single PRs, just to see them in an overview. By the time I do this, I already left the mobile app. It is good to have those options, just an option to "Show all" is missing, like we have for Visibility, Organization and Repository.

**Reply by afitzgerald** [reply]
3 years and still no "show all" option. No way to find the PR to assign to myself for example. 

![image](https://github.com/user-attachments/assets/d933de37-baf7-4a27-8c85-81bad35b73f4)

**Comment by rsatenie** [comment]
This is infuriating all the time. How is this still not implemented is beyond me.
